project_templates/fastapi_safir_app/example/src/example/handlers/external.py

```diff
@@ -0,0 +1,50 @@
+"""Handlers for the app's external root, ``/example/``."""
+
+from fastapi import APIRouter, Depends
+from safir.dependencies.logger import logger_dependency
+from safir.metadata import get_metadata
+from structlog.stdlib import BoundLogger
+
+from ..config import config
+from ..models import Index
+
+__all__ = ["get_index", "external_router"]
+
+external_router = APIRouter()
+"""FastAPI router for all external handlers."""
+
+
+@external_router.get(
+    "/",
+    description=(
+        "Document the top-level API here. By default it only returns metadata"
+        " about the application."
```

Comment:
I just realized that I was doing the opposite in multi-line strings by using "trailing" whitespace rather than leading space. No need to change any of this here, but I just never realized there's more than one way to do this.